connmgr/connmanager.go

```diff
@@ -408,6 +408,14 @@ func (cm *ConnManager) Connect(c *ConnReq) {
 	if atomic.LoadInt32(&cm.stop) != 0 {
 		return
 	}
+
+	//During the time we wait for retry there is a chance that
+	//this connection was already cancelled
```

Comment:
missing spaces between comment slashes and comment body

Reply:
fixed!